dashboard/app/models/reference_guide.rb

```diff
@@ -0,0 +1,70 @@
+require 'fileutils'
+
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  name              :string(255)
```

Comment:
Wondering if calling this display_name might be more clear on its difference from slug? we use display_name on some of our other newer models (CourseVersion, CourseOffering)